aquarius/app/util.py

```diff
@@ -76,6 +77,31 @@ def get_signature_vrs(raw):
     return values
 
 
+def get_signature_bytes(raw):
+    try:
+        wallet = get_aquarius_wallet()
+
+        keys_pk = keys.PrivateKey(wallet.key)
+        message_hash = Web3.solidityKeccak(
+            ["bytes"],
+            [Web3.toHex(Web3.toBytes(text=raw))],
```

Comment:
You don't need both `toHex` and `toBytes` here. Please simplify to just use one or the other. I recommend using `Web3.toBytes(text=message)` for consistency with line 90 below. 
 
```python
>>> from web3.main import Web3
>>> msg = "hello"
>>> Web3.solidityKeccak(["bytes"], [Web3.toHex(Web3.toBytes(text=msg))])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
>>> Web3.solidityKeccak(["bytes"], [Web3.toBytes(text=msg)])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
>>> Web3.solidityKeccak(["bytes"], [Web3.toHex(text=msg)])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
```